app/mailers/claim_mailer.rb

```diff
@@ -1,10 +1,22 @@
 class ClaimMailer < Mail::Notify::Mailer
   def submitted(claim)
+    view_mail_with_claim_and_subject(claim, "Your claim was received")
+  end
+
+  def approved(claim)
+    view_mail_with_claim_and_subject(claim, "Your claim to get your student loan repayments back has been approved. Reference: #{claim.reference}")
+  end
+
+  private
+
+  def view_mail_with_claim_and_subject(claim, subject)
     @claim = claim
+    @display_name = [claim.first_name, claim.surname].compact.join(" ")
```

Comment:
No need for the `compact`, we shouldn't have a completed claim without a first name and surname.